TaskModel.json

```diff
@@ -3010,6 +3010,20 @@
 			"group": 1,
 			"stopsEverywhere": true
 		},
+		{
+			"name": "Eurotunnel Le Shuttle nach %2$s",
+			"service": 10,
+			"descriptions": [
+				"Fahre die Fahrzeuge durch den Eurotunnel nach %2$s.",
+				"Verbinde das Vereinigte Königreich mit Europa.",
+				"Fahre Autos und Busse von %s nach %s."
+			],
+			"stations": [ "XFFE", "🇬🇧ETUK" ],
+			"neededCapacity": [
+				{ "name": "cars", "value": 0 }
```

Comment:
In der Capacity muss noch ein Exchange Factor angegeben werden.

Reply:
Wie sehr schwankt die denn? Ich könnte dann auch die capacity einfach an die Kapazität des Le Shuttle anpassen 🤷‍♀️

Reply:
Es ist jetzt auf 124 festgelegt.
Wobei die Task im Endeffekt eh etwas blöd ist, weil es nur dieses eine Fahrzeug dafür gibt.

Ich glaube, ich mache besser eine Direktvergabe draus...